### PR TITLE
Go mod changes

### PR DESCRIPTION
**Pull request by notzippy**

### CODE REVIEW

File: manual/tool.md [hunk 1]
```diff
@@ -27,12 +27,11 @@ Usage:
...
-      --historic-run-mode  If set the runmode is passed a string not json
-  -X, --build-flags=       These flags will be used when building the application. May be specified multiple times, only applicable for Build, Run, Package, Test commands                           
-
-Help Options:
-  -h, --help               Show this help message
+  -v, --debug                If set the logger is set to verbose
+      --historic-run-mode    If set the runmode is passed a string not json
+      --historic-build-mode  If set the code is scanned using the original parsers, not the go.1.11+
+  -X, --build-flags=         These flags will be used when building the application. May be specified multiple times, only applicable for Build, Run, Package, Test commands
+      --gomod-flags=         These flags will execut go mod commands for each flag, this happens during the build process
```
Comment (ptman) [inline manual/tool.md:34]: execute?

File: manual/organization.md [hunk 2]
```diff
@@ -11,7 +11,32 @@ Revel requires itself and the user application to be installed into a GOPATH lay
 ## Default Layout
 
 Below is the recommended layout of a Revel application, supplemented with
-domain entities and services. 
+domain entities and services. (version 1+)
+ 
+- `my_gocode/`                  - GOPATH root
+    - `go.mod`                      - The go.mod folder
```
Comment (ptman) [inline manual/organization.md:17]: file?

File: tutorial/gettingstarted.md [hunk 3]
```diff
@@ -11,6 +11,8 @@ Before you can use Revel, first you need to [install Go](http://golang.org/doc/i
 - See the official [Go installation guide](https://golang.org/doc/install).
     - [Ubuntu](https://github.com/golang/go/wiki/Ubuntu)
     - [Windows](https://golang.org/doc/install#windows)
+    
+**Because of Default HTTP Server's graceful shutdown, Go 1.8+ is required.**
```
Comment (ptman) [inline tutorial/gettingstarted.md:15]: 1.8 is very old, unsupported, and doesn't support go.mod. Also it doesn't support SameSite: https://github.com/revel/revel/issues/1359

Comment (notzippy) [reply tutorial/gettingstarted.md]: I removed this line as it is no longer relevant

File: manual/tool.md (hunk 1 above)
Comment (notzippy) [reply manual/tool.md]: corrected

**Pull request merged**